bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java

```diff
@@ -265,12 +265,12 @@ public void run() {
             } catch (UnavailableException e) {
                 LOG.error("UnavailableException "
                         + "while replicating fragments", e);
-                waitBackOffTime(rwRereplicateBackoffMs);
                 if (Thread.currentThread().isInterrupted()) {
                     LOG.error("Interrupted  while replicating fragments");
                     shutdown();
                     return;
                 }
+                waitBackOffTime(rwRereplicateBackoffMs);
```

Comment:
We already handled the InterruptedException at line 251. It looks like there want to check if the thread is interrupted after waiting the time. In the waitBackOffTime, it may interrupt the current thread.